rules-reviewed/jboss-eap/websphere/WebSphereMQ.windup.xml

```diff
@@ -0,0 +1,1363 @@
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-24yoad">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeMessage"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the TemporaryQueue class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.TemporaryQueue`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-26root">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeBytesMessage"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the BytesMessage class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.BytesMessage`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-28bvli">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeMapMessage"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the MapMessage class.">
+                    <link description="Javadoc" href="http://www-01.ibm.com/support/knowledgecenter/SSVHEW_6.2.0/com.ibm.mqe.javadoc/java/com/ibm/mqe/jms/package-summary.html"></link>
+                </classification>
+                <hint effort="1">
+                    <message>Replace with Java EE 6 JMS: `javax.jms.MapMessage`.</message>
+                    <link description="Java EE 6 JMS Tutorial" href="https://docs.oracle.com/javaee/6/tutorial/doc/bnceh.html"/>
+                </hint>
+            </perform>
+        </rule>
+        <rule id="websphere-mqe-2Aqqrj">
+            <when>
+                <javaclass references="com.ibm.mqe.jms.MQeObjectMessage"/>
+            </when>
+            <perform>
+                <classification title="WebSphere MQe API." description="A more general, and recommended, approach is to use the ObjectMessage class.">
```

Comment:
Since you link the same MQE documentation in each classification, all of these classifications could easily be replaced by a single rule that matches on the package "com.ibm.mqe"